subt/main.py

```diff
@@ -273,12 +281,33 @@ def follow_wall(self, radius, right_wall=False, timeout=timedelta(hours=3), dist
                 if self.lora_cmd is not None:
                     # the "GoHome" command must be accepted only on the way there and not on the return home
                     if dist_limit is None and self.lora_cmd == LORA_GO_HOME_CMD:
-                        print('LoRa cmd - GoHome')
+                        print(self.time, 'LoRa cmd - GoHome')
                         self.lora_cmd = None
                         break
                     if self.lora_cmd == LORA_STOP_CMD:
-                        print('LoRa cmd - Stop')
-                        break
+                        print(self.time, 'LoRa cmd - Stop')
+                        # this command does not clean "self.lora_cmd" intentionally
+                        # so that repeated call is also stopped
+                        # maybe it should raise EmergencyStop instead?!
+                        raise EmergencyStopException()
+                    if self.lora_cmd == LORA_PAUSE_CMD:
```

Comment:
Aren't the lora commands mutually exclusive?

Reply:
OK, except the first case now used `elif`